### PR TITLE
Fix Commit Button

### PR DESCRIPTION
**Pull request by minhle10**
Disabled the 'Committed' button.

### CODE REVIEW

File: src/components/AlgoGovernanceContent.jsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ export default function AlgoGovernanceContent() {
             setMaxBal(value.balance)
           }}
           // variant ={true}
-          disabled = {false}
+          disabled = {true}
```
Comment (mccabe-david) [inline src/components/AlgoGovernanceContent.jsx:86]: This should say value.committed === value.collateral instead of just being disabled by default

**Review by mccabe-david — DISMISSED** [review]

File: src/components/AlgoGovernanceContent.jsx (hunk 1 above)
Comment (mccabe-david) [reply src/components/AlgoGovernanceContent.jsx:86]: People may add more collateral later if they so desire.

**Comment by ElianaTroper** [comment]
image

There seems to be some issue occurring

**Comment by minhle10** [comment]
> image
> 
> There seems to be some issue occurring

I think I see why: value.committed is undefined because we pull the committed value from firebase now. I'll fix it.

**Comment by minhle10** [comment]
Actually, hold up value.collateral is the one that is undefined.

**Comment by minhle10** [comment]
Alright, I just fixed it. Sorry about that, I should have checked.

**Review by ElianaTroper — APPROVED** [review]

**Pull request merged**